app/helpers/user_api.rb

```diff
@@ -26,13 +26,18 @@ def self.get_user_info_pii(user_ids)
     HTTParty.post(competitor_info_path, headers: { WCA_API_HEADER => self.wca_token }, body: { ids: user_ids.to_a })
   end
 
-  def self.can_compete?(user_id)
+  def self.can_compete?(user_id, competition_start_date)
```

Comment:
I think it's fine to pass in the comp start date here, but this did make me wonder if we should bring in the banned competitor logic to the registration_checker, which already has the competition info loaded in an instance variable.

Reply:
I think it's fine, the monolith also adds the competition_start_date in its function